### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jgalan**
Lets merge development to master. Then we can remove `development` and just do pull-requests to master.

### CODE REVIEW

**Comment by jgalan** [comment]
Plz, could you approve this PR? After approval we will avoid having a `development` branch, and we will push directly to master.

**Review by nkx111 — APPROVED** [review]

**Review by juanangp — APPROVED** [review]

**Review by lobis — APPROVED** [review]

**Pull request merged**
